### PR TITLE
Add move.nix to flake manual

### PR DESCRIPTION
**Pull request by macalinao**

### CODE REVIEW

**Comment by ncfavier** [comment]
This is an index of flakes that are useful to the community, not a place to advertise your company, *much less* your random web3 thing nobody asked for.

**Comment by macalinao** [comment]
Hey @ncfavier -- I didn't make Move! I've just built a lot of tooling around it, and to my knowledge there are a lot of competing businesses building on top of it.

IOHK is also listed in here. I would argue that this flake should be in here for the same reason.

Thanks and I hope you can understand.

**Comment by ysndr** [comment]
I'm going to reopen this because there is actually no manifested guideline of what goes into the flake search and what not (https://github.com/NixOS/nixos-search/issues/486).

Its not a guarantee that this will be merged but 

a) @MatthewCroughan has a point in https://github.com/NixOS/nixos-search/pull/510
b) Evidently this is currently not necessarily "an index of flakes that are useful for the community" 
  - ie. which community? (nix or general open source)
  - bitcoin-nix, pinpox/nixos

I see a point in inclusivity as well as I see it in excluding commercial projects (I cant tell if moveco s but it sounds like it)
I'm not quite a proponent for web3 stuff either personally, but I would like to grant this a chance in a proper discussion.
Once we have a guideline, this might entail _removing_ specific flakes from the current registry or other pending ones to be added.

tl;dr

Reopend to as pending
Waiting for an outcome on https://github.com/NixOS/nixos-search/issues/486

**Comment by macalinao** [comment]
> I'm going to reopen this because there is actually no manifested guideline of what goes into the flake search and what not (https://github.com/NixOS/nixos-search/issues/486).
> 
> 
> 
> Its not a guarantee that this will be merged but 
> 
> 
> 
> a) @MatthewCroughan has a point in https://github.com/NixOS/nixos-search/pull/510
> 
> b) Evidently this is currently not necessarily "an index of flakes that are useful for the community" 
> 
>   - ie. which community? (nix or general open source)
> 
>   - bitcoin-nix, pinpox/nixos
> 
> 
> 
> I see a point in inclusivity as well as I see it in excluding commercial projects (I cant tell if moveco s but it sounds like it)
> 
> I'm not quite a proponent for web3 stuff either personally, but I would like to grant this a chance in a proper discussion.
> 
> Once we have a guideline, this might entail _removing_ specific flakes from the current registry or other pending ones to be added.
> 
> 
> 
> tl;dr
> 
> 
> 
> Reopend to as pending
> 
> Waiting for an outcome on https://github.com/NixOS/nixos-search/issues/486 
> 
> 
> 
> 

Just to be clear--movingco is not a commercial project. In fact, I'm happy to give the rights to the move.nix repo to NixOS (actually, I'm fine giving away the entire GitHub organization).

This is a flake that just builds open source software, most of which is blockchain related.

And I would argue that the world _needs_ Move-- over $50B have been hacked from smart contracts, and Move reduces smart contract risk greatly. While it may not affect you personally, it has affected many people in seriously detrimental ways. Combining Move with Nix adds another level of safety by encouraging the use of good CI practices and by making binaries more accessible.

Yes you may disagree with the idea of crypto in general, but the solution is not to make software more crappy. The solution is to move the world over to Nix, NixOS, and Apache2 open source.

**Comment by raboof** [comment]
This repo hasn't been updated in 3 years - is it still relevant?

**Comment by raboof** [comment]
(closing for new, feel free to comment if you'd like to discuss reopening)
